python/blocks.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+
+"""
+    blocks.py
```

Comment:
It does not look like you've auto-generated this file. Please see: https://github.com/wikimedia/MediaWiki-Action-API-Code-Samples#contributing-code-samples.

Reply:
I did autogenerated this file. When I saw the other pages you improved none of the codes there had that autogenerated comment, so I deleted it, if you wish I can edit it to include the comment.